mapping.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
@willdoran @rowasc is this actually used?

Reply:
No this is redundant.